h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -3836,7 +4151,7 @@ private void checkCoeffsBounds() {
       int coeffsLen = bc._betaLB.length;
       for (int index=0; index < coeffsLen; index++) {
         if (!(coeffs[index] == 0 || (coeffs[index] >= bc._betaLB[index] && coeffs[index] <= bc._betaUB[index])))
-          throw new H2OFailException("GLM model coefficient" + coeffs[index]+" exceeds beta constraint bounds.  Lower: "
+          throw new H2OFailException("GLM model coefficient " + coeffs[index]+" exceeds beta constraint bounds.  Lower: "
```

Comment:
It would be usefull to print all exceeds beta constraints and after that throw H2OFailException. WDYT?

Reply:
I love the idea and will incorporate it.